Makefile

```diff
@@ -0,0 +1,39 @@
+DEPLOY_DATE = $(shell date +%s)
+
+update-resources:
+	git submodule update
+	  cd resource-types && \
+	  git checkout master && \
+	  git pull
+	git add resource-types && \
```

Comment:
looks like you cd into resource-types then try to add resource-types folder without goin' up a directory? just tried locally and it had a problem with that

Reply:
I remember it was working on my side. Probably because `git submodule update` wasn't run in your repo?
I will give it another shot.

Reply:
i might just be crazy (i did just copy and paste the instructions)

Reply:
yeah it worked for me :D my bad